registry/src/pages/createNamespace.js

```diff
@@ -15,6 +17,33 @@ const NamespaceForm = () => {
   const message = useSelector((state) => state.createNamespace.message);
   const statuscode = useSelector((state) => state.createNamespace.statuscode);
 
+  useEffect(() => {
```

Comment:
This seems to work not so well for me.

When I click submit multiple time. I don't see any toast message. This I think is related to same state being emitted consecutively. So, that is why it is not getting reflected. Since statusCode is not changing.

Reply:
will add a redirect to the dashboard after the toast completion.

Reply:
You might not want to redirect if `statusCode` is not `200` (that means request resulted in some error). You might still want the user to retry again by staying on that page.

Redirection makes sense to me only when request was a success and you no longer want to stay on namespace creation page.

That means, I think it would be great to fix that state issue.

Reply:
Sure.